Automattic/newspack-plugin
# Fine-tune and update dashboard/checklist options

### All Submissions:






### Changes proposed in this Pull Request:



This PR brings the dashboard in line with the latest designs (except images). All the checklists are present, but the ones that haven't been built yet are disabled. I've removed the demo checklist, since it's not needed now that we have a real checklist available.

### How to test the changes in this Pull Request:

1. `npm run clean; npm run build:webpack`
2. Navigate to the Newspack Dashboard. You should see this:
![localhost_wp-admin_admin php_page=newspack](https://user-images.githubusercontent.com/7317227/60198986-bcb1dc80-97f7-11e9-8ab3-a90db1d80985.png)
3. You should be able to go to the Reader Revenue checklist and not the other checklists.

### Other information:
